partiql-planner/src/main/kotlin/org/partiql/planner/internal/util/DateTimeUtils.kt

```diff
@@ -33,7 +33,7 @@ internal object DateTimeUtils {
     internal fun parseTime(input: String): LocalTime {
         val matcher: Matcher = TIME_PATTERN.matcher(input)
         if (!matcher.matches()) throw DateTimeException(
-            "Expect TIME Format to be in HH-mm-ss.ddd+[+|-][hh:mm|z], received $input"
+            "Expect TIME Format to be in HH:mm:ss.ddd+[+|-][hh:mm|z], received $input"
```

Comment:
(self-review) these error messages were slightly off. Should use colons rather than dashes for the time part delimiters